### PR TITLE
APPS-1607 retry errorstrategy - last retry ends in fail state

### PR DESCRIPTION
**Pull request by mhrvol**
https://staging.dnanexus.com/projects/GG76K7j01xy3fZ7zFZ94YBY8/monitor/job/GP9vZ6Q01xy7vgv4k8868x3B - example run with maxErrors reached

Do not merge before updating all the assets!

### CODE REVIEW

File: src/python/dxpy/nextflow/nextflow_utils.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def get_nextflow_assets(region):
             "record-GP5B1FQJ1pz6gB19bv9BYkPP",
             "record-GKv18b0Jz9YV9vqx9j9GGq05"),
         "aws:us-east-1": (
-            "record-GP59VV80zXP4843ffZ0Y7fv2",
+            "record-GP9vPf00p5pX5Vzq8jjYP8Vz",
```
Comment (mhrvol) [inline src/python/dxpy/nextflow/nextflow_utils.py:93]: This will need record change for all regions.

**Review by commandlinegirl — APPROVED** [review]
This LGTM but could you describe in a bit more detail this solution in the JIRA ticket for a future record.

**Comment by mhrvol** [comment]
nextflow python3 tests: https://jenkins-vstg.internal.dnanexus.com/job/dx-toolkit/job/dxpy-branch-integration-tests-on-staging-20.04-python3/331/

**Comment by mhrvol** [comment]
Merging - updates since last approval are only in test, nextaur version update and nextaur assets update

**Pull request merged**